x-pack/test/functional/services/transform/wizard.ts

```diff
@@ -798,6 +799,72 @@ export function TransformWizardProvider({ getService, getPageObjects }: FtrProvi
       );
     },
 
+    async assertTransformNumFailureRetriesInputExists() {
+      await testSubjects.existOrFail('transformNumFailureRetriesInput');
+      expect(await testSubjects.isDisplayed('transformNumFailureRetriesInput')).to.eql(
+        true,
+        `Expected 'Number of retries failure' input to be displayed`
+      );
+    },
+
+    async assertNumFailureRetriesValue(expectedValue: string) {
+      await this.assertTransformNumFailureRetriesInputExists();
+      const actualValue = await testSubjects.getAttribute(
+        'transformNumFailureRetriesInput',
+        'value'
+      );
+      expect(actualValue).to.eql(
+        expectedValue,
+        `Transform num failure retries text should be '${expectedValue}' (got '${actualValue}')`
+      );
+    },
+
+    async assertTransformNumFailureRetriesErrorMessageExists(expected: boolean) {
+      const row = await testSubjects.find('transformNumFailureRetriesFormRow');
+      const errorElements = await row.findAllByClassName('euiFormErrorText');
+
+      if (expected) {
+        expect(errorElements.length).greaterThan(
+          0,
+          'Expected Transform num failure retries to display error message'
+        );
+      } else {
+        expect(errorElements.length).eql(
+          0,
+          'Expected Transform num failure retries to not display error message'
+        );
+      }
+    },
+
+    async setTransformNumFailureRetriesValue(value: string, expectedResult: 'error' | string) {
+      await retry.tryForTime(5000, async () => {
+        await testSubjects.setValue('transformNumFailureRetriesInput', value);
+        if (expectedResult !== 'error') {
+          await this.assertTransformNumFailureRetriesErrorMessageExists(false);
+          await this.assertNumFailureRetriesValue(expectedResult);
+        } else {
+          await this.assertTransformNumFailureRetriesErrorMessageExists(true);
+        }
+      });
+    },
+
+    async assertTransformNumFailureRetriesSummaryValue(expectedValue: string) {
+      await retry.tryForTime(5000, async () => {
+        await testSubjects.existOrFail('transformWizardAdvancedSettingsNumFailureRetriesLabel');
+        const row = await testSubjects.find(
+          'transformWizardAdvancedSettingsNumFailureRetriesLabel'
+        );
+        const actualValue = await (
+          await row.findByClassName('euiFormRow__fieldWrapper')
+        ).getVisibleText();
+
+        expect(actualValue).to.eql(
+          actualValue,
```

Comment:
Need to check against the expected value:
```suggestion
          expectedValue,
```

Reply:
Haha thanks for catching that! Fixed here https://github.com/elastic/kibana/pull/135486/commits/9258c68f867e320b2663f2f341d7d1c65f00d1af